src/pages/HomePage.tsx

```
@@ -15,168 +15,135 @@ const HomePage = () => {
-    {experts && experts.map((expert)=>(
-      <div>
-        <ExpertCard expert={expert}/>
-      </div>
-    ))}
+  <div className="grid grid-cols-1 sm:grid-cols-2 md:grid-cols-4 gap-6 py-6">
+    {cards &&
+      cards.map((card) => (
+        <div key={card.id}>
+          <Card card={card} />
+        </div>
+      ))}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)